saleor/attribute/utils.py

```diff
@@ -19,118 +19,154 @@
 T_INSTANCE = Union[Product, ProductVariant, Page]
 
 
-if TYPE_CHECKING:
-    from .models import AttributePage, AttributeProduct, AttributeVariant
-
-
 def associate_attribute_values_to_instance(
-    instance: T_INSTANCE,
-    attribute: Attribute,
-    *values: AttributeValue,
-) -> AttributeAssignmentType:
-    """Assign given attribute values to a product or variant.
-
-    Note: be aware this function invokes the ``set`` method on the instance's
-    attribute association. Meaning any values already assigned or concurrently
-    assigned will be overridden by this call.
-    """
```

Comment:
I would suggest not removing this docsting. It contains important information that the attribute values are overridden